.vale.ini

```diff
@@ -0,0 +1,14 @@
+StylesPath = .github/styles
+
+Vocab = Docs
+
+[*.asciidoc]
+BasedOnStyles = Google, Microsoft, write-good, Vale
+
+Google.Headings = NO
+Google.WordList = NO
+Microsoft.GeneralURL = NO
+Microsoft.Contractions = NO
+Microsoft.HeadingAcronyms = NO
+write-good.TooWordy = NO
+Vale.Spelling = NO
```

Comment:
What does this disable? I’m wondering if we should instead start to update the Vocab instead?

Reply:
Ah yeah, that was a too-quick fix. I'll add the words to the vocabulary.